src/app/login-signup/register/register.component.html

```diff
@@ -1 +1,30 @@
-<p>register works!</p>
+<div class="container">
+    <!--Registration form-->
+    <form [formGroup]="registrationForm">
+        <h3>Register Here</h3><br>
+        <div class="input-fields">
+            <!--Email-->
+            <div>
+                <label for="username">Email</label><br>
```

Comment:
Remove all br tag and add proper styling

Reply:
Changes are done